### PR TITLE
use old github library

### PR DESCRIPTION
**Pull request by nickpell**
there's still an issue to be worked out with the new one

### CODE REVIEW

**Comment by orangejenny** [comment]
Looks reasonable, though I don't have context...is this reverting a particular set of PRs? I thought it'd be a direct revert of https://github.com/dimagi/commcare-hq/pull/18232 but doesn't look like it.

File: fab/utils.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import re
 from getpass import getpass
 
-from github import Github, InputGitAuthor
+from github3 import login
```
Comment (orangejenny) [inline fab/utils.py:14]: Does github3 need to go back into requirements.txt, a la https://github.com/dimagi/commcare-hq/pull/18232/files#diff-444e80f30bd2f1e25b779966c9621b9bL90 ?

Comment (nickpell) [reply fab/utils.py:14]: yep: https://github.com/dimagi/commcare-hq/pull/18646

**Comment by nickpell** [comment]
@orangejenny There were a few PRs that changed code, so I included the reverted commits from all of them

**Review by orangejenny — APPROVED** [review]

**Review by kaapstorm — APPROVED** [review]

**Pull request merged**